genmc-sys/cpp/include/MiriInterface.hpp

```diff
@@ -234,6 +245,14 @@ struct MiriGenmcShim : private GenMCDriver {
      * indices, since GenMC expects us to do that.
      */
     std::vector<Action> threads_action_;
+
+    /**
+     * Map of already used annotation ids (e.g., for mutexes).
+     * FIXME(GenMC): Ensure that these are consistent with how GenMC expects them once Miri supports
+     * multithreading in GenMC mode.
+     */
+    std::unordered_map<uint64_t, ModuleID::ID> annotation_id {};
```

Comment:
Judging from the rest of the code here, this uint64_t is the address of the mutex, right? That must be explained in a comment here.